shared/src/commonMain/kotlin/me/saket/press/shared/editor/EditorPresenter.kt

```diff
@@ -81,7 +81,15 @@ class EditorPresenter(
         .take(1)
         .flatMapCompletable { (existingNote) ->
           when (existingNote) {
-            null -> noteRepository.create(newOrExistingId, NEW_NOTE_PLACEHOLDER)
+            null -> {
+              val note = (openMode as NewNote).note
+              val content = if (note.isBlank()) {
+                NEW_NOTE_PLACEHOLDER
+              } else {
+                note
+              }
```

Comment:
When `note.preFilledNote` becomes nullable, this can be reduced to 

```kotlin
val content = note.preFilledNote ?: NEW_NOTE_PLACEHOLDER
```

Reply:
Actually, `openMode` may or may not be of type `NewNote`. This will crash if an existing note was removed from the database while this screen was opened.

Reply:
Is there a way I can trigger this scenario? Because we cannot remove a note when screen is opened right?

Reply:
For now I just changed it to 
```kotlin
val note = openMode as? NewNote
val content = note?.preFilledNote ?: NEW_NOTE_PLACEHOLDER
```

Reply:
> Is there a way I can trigger this scenario? Because we cannot remove a note when screen is opened right?

Yep, in a multi-device world it's possible to delete the note from another device. It's also mentioned in a comment on line 76, but I guess they're easy to miss.